packages/bits/src/lib/sorter/sorter.component.html

```diff
@@ -22,7 +21,6 @@
     <div #popupArea></div>
 
     <nui-overlay #overlay
-                 [role]="'menu'"
                  [toggleReference]="toggleRef"
```

Comment:
should the roleAttr be specified here?

Reply:
No, because of `nui-menu-popup` inside `nui-overlay`. `nui-menu-popup` under the hood has the role="menu"